domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectTestBuilder.kt

```diff
@@ -19,17 +19,15 @@ object InteractionObjectTestBuilder {
     return InteractionObject.newBuilder().setNormalizedString(value).build()
   }
 
-  fun createListOfSetsOfHtmlStrings(listOfStringList: List<StringList>): InteractionObject {
+  fun createListOfSetsOfHtmlStrings(vararg items: List<String>): InteractionObject {
```

Comment:
We don't need to change this.

Reply:
In this function, only one attribute could be passed. I'm getting errors while passing multiple attributes.

Reply:
Tell about the error you are facing

Reply:
[https://user-images.githubusercontent.com/56022539/97670141-9e26ba00-1aab-11eb-87d7-75d655fe74cc.JPG
](url) 
Error which I'm facing: Too many arguments for public final fun createListOfSetsOfHtmlStrings(listOfStringList: List<[ERROR: StringList]>): [ERROR: InteractionObject] defined in org.oppia.android.domain.classify.InteractionObjectTestBuilder.

Getting this after undoing the changes I did earlier. What changes should I do in fun createListOfSetsOfHtmlStrings in InteractionObjectTestBuilder.kt in order to resolve this?

Reply:
![oppia1](https://user-images.githubusercontent.com/56022539/97671797-edbab500-1aae-11eb-90f0-49b10350f223.JPG)
The above mentioned link was not working so please use this one.

Reply:
@anandwana001 @ARJUPTA PTAL

Reply:
@Aarkin7 
See the methods argument

```kotlin
createListOfSetsOfHtmlStrings(listOfStringList: List<StringList>)
```

Here, we need to pass a list of `StringList`. 

To get the string list, we have a method `createHtmlStringList`. List of these string list, you had to pass in this method which asks for list of string list.